src/basic/log.c

```diff
@@ -1449,20 +1449,34 @@ void log_received_signal(int level, const struct signalfd_siginfo *si) {
         assert(si);
 
         if (pid_is_valid(si->ssi_pid)) {
-                _cleanup_free_ char *p = NULL;
+                _cleanup_free_ char *p = NULL, *u = NULL;
 
                 (void) get_process_comm(si->ssi_pid, &p);
 
+                (void) cg_pid_get_unit(si->ssi_pid, &u);
+
                 log_full(level,
-                         "Received SIG%s from PID %"PRIu32" (%s).",
+                         "Received SIG%s from PID %"PRIu32" (%s)%s%s%s.",
                          signal_to_string(si->ssi_signo),
-                         si->ssi_pid, strna(p));
+                         si->ssi_pid, strna(p), u ? " (unit " : "", u ? u : "", u ? ")" : "");
+
         } else
                 log_full(level,
                          "Received SIG%s.",
                          signal_to_string(si->ssi_signo));
 }
 
+void log_caller(pid_t pid, const char *comm, const char *caller, pid_t ppid, const char *comm_parent, const char *method) {
```

Comment:
changes in this file are not necessary, drop them all - again, only move log_caller as bus_log_caller and use it without adding or removing things from it, apart from the minimum necessary to transfer it

Reply:
The `log_caller` is indeed a leftover after `log_received_signal` was equipped with logging unit of the caller.